src/main/java/io/github/dsheirer/record/wave/ComplexBufferWaveRecorder.java

```diff
@@ -89,8 +90,8 @@ public void start()
                 sb.append("_");
                 sb.append(TimeStamp.getTimeStamp("_"));
                 sb.append(".wav");
-
-                mFile = Paths.get(sb.toString());
+                String cleaned = StringUtils.replaceIllegalCharacters(sb.toString());
```

Comment:
Could you please move this clean method up into the RecorderManager.getBasebandRecorder() method so that linux filepaths don't get corrupted:

For example:

        StringBuilder sb = new StringBuilder();
        sb.append(getRecordingBasePath());
        sb.append(File.separator).append(StringUtils.replaceIllegalCharacters(channelName)).append("_baseband");
        return new ComplexBufferWaveRecorder(BASEBAND_SAMPLE_RATE, sb.toString());